pytentiostat/routines.py

```diff
@@ -23,13 +23,14 @@ def _load_arduino():
     ports = list(serial.tools.list_ports.comports())
     n_arduinos = 0
     for p in ports:  # Checking for Arduino Unos connected
-        if "Arduino Uno" in p.description:
+        if re.search("tty|Arduino Uno|COM",p.description) is not None:
             com = p.device
             n_arduinos += 1
+            print("INFO: Arduino Uno found at port.\n".format(com))
```

Comment:
This increases n_arduinos by 1 for every COM device connected to the computer. It needs to be specific for finding arduinos only. Could you tell us what string does your system return for `p.description` when `p` is the arduino? Maybe we can help here.

Reply:
Yes of course, the system return "ttyACM0". I guess that is because I'm using a Linux machine.

Reply:
Also, we should test the scenario if one of the USBs has a device connected at the same time as an arduino, especially if the port would be listed before the one at the arduino. For instance if we want to use the check description case if opening serial connection or check description fails, should try another port that matched the search. Maybe search should make a list of possibly compatible devices.

Reply:
After looking on google, it looks like the Arduino Uno description will usually show up in linux as "ttyACMX". On windows it shows up as "Arduino Uno (COMX)". Maybe the best thing to do here is...
`if re.search("ttyAMC|Arduino Uno", p.description) is not None:
     ...`

Reply:
I guess last one is mac case?